src/aui-form-validator/js/aui-form-validator.js

```diff
@@ -840,17 +840,7 @@ var FormValidator = A.Component.create({
                     }
                 }
                 else if (!field.val()) {
-                    namedFieldNodes.each(
-                        function(node, index, nodeList) {
-                            node.removeClass('errorClass');
-                            node.removeAttribute('aria-invalid');
-                        }
-                    );
-
-                    if (fieldContainer) {
```

Comment:
I just saw that `resetFieldCss` would reset the class for this `fieldContainer` too, but I don't see where the `removeAttribute` call for the container is done after this change. I saw that you added one for the named fields in another commit, but don't see it for this container.